pontoon/checks/libraries/not_stored.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
```

Comment:
I think we should call this file `no**n**_stored.py` instead.

Reply:
ok